networkx/drawing/tests/test_pylab.py

```diff
@@ -62,6 +62,18 @@ def test_draw_shell_nlist():
             pass
 
 
+def test_draw_bipartite():
```

Comment:
Hmm, can this just go in the `test_draw` function up there?

Reply:
The reason is the `barbell_graph` used in the `test_draw` example isn't bipartite, which doesn't work with the default arguments!